docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,47 @@
+You can also check out our [existing game samples](additional-game-samples) or the [live demo of our validation tool](https://play.regression.gg/validations-demo).
 
-* A single account can run at least 10 bots concurrently
-* The time from requesting a bot to be started to the time it connects to Unity will take less than 10 seconds if the bot is relying on dependencies that are common to most bots. This does not cover bots that may have lots of library dependencies, which may take longer.
+## Our Mission
 
-## Implemented Features
+Our mission is to transform the landscape of game development by tackling one of its most persistent challenges: automated testing.
 
-The table below details and links to documentation for the features implemented on our platform.
+Our tools and standards will make effective end-to-end automated testing of games a reality, in a way that actually saves you time and pain. We imagine a future where automated tests handle most functional testing scenarios, freeing up valuable time for developers to focus on the more challenging and creative aspects of game development. Regression Games is making this possible by combining old school algorithms, cutting-edge machine learning techniques, and a focus on the developer experience.
 
-| **Feature**                    | **Description**                                                                                                   | **Documentation**                                         |
-|--------------------------------|-------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------|
-| Unity C# SDK                   | A C# SDK for implementing bots locally in Unity.                                                                  | [Link](unused-content/creating-bots/csharp/adaptive-bots) |
-| Bot Code Generation            | Create bots with low-code tools such as behavior trees powered by GPT.                                            | [Link](unused-content/creating-bots/csharp/agent-builder) |
-| Platform and Language Agnostic | Our protocol permits bots to eventually be written in multiple engines and languages.                             | _Docs coming soon_                                        |  
+Our vision is to build tools with the community that actually help you be more productive. There are three core principles driving the development of this framework:
+- **Ease of use** - Developers should be able to integrate and create new tests with minimal effort.
+- **Reliability** - In the dynamic world of games, reliability is key. Our framework is designed to handle the unexpected and give you the tools to create robust, dependable tests.
+- **Data driven** - Our framework collects comprehensive data, empowering you to build smarter automations and create more effective tests.
```

Comment:
I feel that `Data driven` is the least punchy of these principles. Trying to think of an alternative 🤔

Reply:
Open to suggestions!